composer.json

```
@@ -5,9 +5,9 @@
     "license": "MIT",
     "require": {
         "php": "^7.3",
```

If you would like to support PHP 8, it should be added also here:
```suggestion
        "php": "^7.3|^8.0",
```